src/KernelStrategy.sol

```diff
@@ -92,6 +92,9 @@ contract KernelStrategy is Vault {
 
         FeeStorage storage fees = _getFeeStorage();
         fees.baseWithdrawalFee = baseWithdrawalFee;
+
+        StrategyStorage storage strategyStorage = _getStrategyStorage();
```

Comment:
We need to update the deploy scripts to ensure this is set correctly for respective vaults